JHUGenerator/mod_Parameters.F90

```diff
@@ -221,11 +221,11 @@ MODULE ModParameters
 !=====================================================
 !constants
 real(8), public            :: M_Top   = 173.2d0   *GeV      ! top quark mass
-real(8), public            :: Ga_Top  = 2.0d0     *GeV      ! top quark width
+real(8), public            :: Ga_Top  != 2.0d0     *GeV      ! top quark width
```

Comment:
Experimental tests like these should be removed before any merging can happen.

Reply:
not really experimental. I calculated the top width from the top mass and the electroweak parameters further down below when initailiazing the EW parameters in order to have consistent branching ratios. I left the commented line in there in case someone wants to put own favorite values.

Reply:
I understand what is done below, but that is not what the conventions of JHUGen are. We leave Ga_W and Ga_Top to be user-definable quantities just like any other constant like cos thetaW. Note that different EW constant schemes different conditions, and we decided not to impose any one of them on the user; those computations would need to be done by them outside of the generator configuration. What you could do is to write the formula for Ga_W and Ga_Top in the documentation in order to match MG.

There is also a blank line change that needs to be removed.

Reply:
(I meant write down the formula in the Manual, not in code.)

Reply:
Alright. I will edit the code accordingly. Cheers